service/samples/registry_bulk_identity_sample.js

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+var iothub = require('azure-iothub');
+var azureStorage = require('azure-storage');
+
+var iothubConnectionString = process.env.IOTHUB_CONNECTION_STRING;
+var storageConnectionString = process.env.STORAGE_CONNECTION_STRING;
+
+var registry = iothub.Registry.fromConnectionString(iothubConnectionString);
+var blobSvc = azureStorage.createBlobService(storageConnectionString);
+
+var startDate = new Date();
+var expiryDate = new Date(startDate);
+expiryDate.setMinutes(startDate.getMinutes() + 100);
+startDate.setMinutes(startDate.getMinutes() - 100);
+
+var inputSharedAccessPolicy = {
+  AccessPolicy: {
+    Permissions: 'rl',
+    Start: startDate,
+    Expiry: expiryDate
+  },
+};
+
+var outputSharedAccessPolicy = {
+  AccessPolicy: {
+    Permissions: 'rwd',
+    Start: startDate,
+    Expiry: expiryDate
+  },
+};
+
+var inputContainerName = 'importcontainer';
+var outputContainerName = 'exportcontainer';
+var deviceFile = 'devices.txt';
+
+blobSvc.createContainerIfNotExists(inputContainerName, function (error) {
+    if(error) {
```

Comment:
Let's keep the indenting consistent. 2 spaces.

Reply:
Same as the below comment, we can talk about changing both the existing sample and this one.  I will close this comment

---
In reply to: [388026930](https://github.com/Azure/azure-iot-sdk-node/pull/709#discussion_r388026930) [](ancestors = 388026930)